data.yml

```diff
@@ -40,6 +40,12 @@
   author_website: http://ethanfrei.com
   description: A smart lamp which displays the current phase of the moon.
   image: http://ethanfrei.com/img/2021/moon-lamp-2.jpg
+- name: Rusty mouse
+  website: https://github.com/brainstorm/bbtrackball-rs
+  author: Roman Valls Guimera and Josh Johnson
+  author_website: https://blogs.nopcode.org/brainstorm
+  description: A simple, interrupt-driven trackball. An stm32f04 is wired to a blackberry trackball and reads pulses from its 4 hall sensors, works on most operating systems.
```

Comment:
Uhm, what is a "stm32f04"?

Reply:
Argh, sorry, STM32F042:

https://github.com/brainstorm/bbtrackball-rs/blob/master/software/stm32f04/Cargo.toml#L21

Shall I open a PR for that missing "2"? :-S

Reply:
I think it would be nice to get the important details right. I'd probably reword the whole description. To an innocent bystander it's likely not clear what this is all about.

Reply:
Very fair point! How's this de-nerdification?:

> A simple USB trackball written in Rust's RTIC (Real-Time Interrupt-driven Concurrency). The code can be reused for many other mouse-like Human Interaction Devices (HID). Works well on PC and Mac computers.

Reply:
That is a great description!